app/pandas.ipynb

```diff
@@ -458,36 +416,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "s = pd.Series([\"numpy\", \"pandas\", \"matplotlib\"])"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# make all words in uppercase"
-   ],
+   "source": "s = s.str.upper()",
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# get length of each word"
-   ],
+   "source": "s = s.str.len()",
```

Comment:
This operation overwrites the Series with its lengths. If you need the original data later, consider storing the lengths in a new variable.